ui/src/__tests__/pages/AllStories.test.ts

```diff
@@ -135,14 +166,28 @@ describe("allstories table", () => {
     it("should search a story by string", () => {
       const searchedVal = "City1";
       const action: Action = {
-        type: "HANDLE_SEARCH",
-        data: searchedVal,
+        type: "HANDLE_SEARCH/FILTER",
+        data: { ...INIT_STATE, search: searchedVal },
       };
       expect(INIT_STATE.search).toBe("");
       const newState = allStoriesReducer(INIT_STATE, action);
       expect(newState.search).toBe(searchedVal);
     });
 
+    it("should filter by tag1", () => {
```

Comment:
Since `HANDLE_SEARCH/FILTER` also changes the `visibleTableState` etc. in r`equestSearchAndFilter() `you can also test those change properly in this function